Formula/groovy@2.rb

```diff
@@ -0,0 +1,32 @@
+class GroovyAT2 < Formula
+  desc "Java-based scripting language"
+  homepage "https://www.groovy-lang.org/"
+  url "https://dl.bintray.com/groovy/maven/apache-groovy-binary-2.5.9.zip"
+  sha256 "fea7dc321a3029c47ffa4aa165055d2bcc78bc280fac4e70ac131c717e45b89b"
+
+  bottle :unneeded
+
+  keg_only :versioned_formula
+
+  # Groovy 2.5 requires JDK8+ to build and JDK7 is the minimum version of the JRE that we support.
+  depends_on :java => "1.7+"
+
+  def install
+    # Don't need Windows files.
+    rm_f Dir["bin/*.bat"]
+
+    libexec.install "bin", "conf", "lib"
+    bin.install_symlink Dir["#{libexec}/bin/*"] - ["#{libexec}/bin/groovy.ico"]
+  end
+
+  def caveats
+    <<~EOS
+      You should set GROOVY_HOME:
+        export GROOVY_HOME=#{opt_libexec}
+    EOS
+  end
```

Comment:
I think this caveat needs to be removed from both this Formula, and the stable groovy formula. This is not a caveat specific to homebrew. Groovy always needs to have `GROOVY_HOME` set, no? Please remove unless I am mistaken.